src/dice.js

```diff
@@ -1,71 +1,84 @@
-dice = {
-	version: "0.7.0",
+var dice = {};
```

Comment:
This block could be rewritten as:

```javascript
var dice = {
    parse: require('./parse').parse,
    eval: require('./evaluate').eval,
    version: require('../package').version
}
```

Additionally, it might be possible to simply export the function, as opposed to an object with a 'parse' function from both `parse` and `evaluate`. I'll comment on those independently if that's the case.

One thing to draw attention to: the lack of `.js` in the requires, and requiring `package.json` to get the version. The first is convention, while the second just makes sure you don't have to update version in more than once place.